src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Image.cs

```diff
@@ -155,5 +157,34 @@ public async Task When_Image_Is_Loaded_From_URL()
 
 			Assert.IsTrue(img.ActualHeight > 0);			
 		}
+      
+#if __IOS__
```

Comment:
Does it fail for the other targets? The code you changed is altering the common behavior, it'll be interesting to know if it works.

Reply:
Yes, it fails for other. 
After investigating issue #7288 i changed the PR to draft, as one solution will impact other. Probably it will better to solve both with the same PR.